README.md

```diff
@@ -1,3 +1,16 @@
 # Training Repo
 
 [TODO: Write 400 words for the description of the repo. Answer: What are you expecting, and how is it going.]
```

Comment:
This line should be removed. It was there as a placeholder for the content you just added.